src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml.cs

```diff
@@ -197,7 +207,7 @@ private void Adornment_KeyDown(object sender, KeyEventArgs e)
                     // loop focus back to the first item that is focusable.
                     FrameworkElement lastItem = _viewModel.IsExpanded
```

Comment:
@veler can you hook up Ctrl+Space to get suggestions?
`case Key.Space:`
`if (Keyboard.Modifiers == ModifierKeys.Ctrl)`